lib/reports/resolvers.ts

```diff
@@ -214,7 +214,7 @@ export const generateApplicationsReport: Resolver<
       return {
         ...application,
         dateOfBirth: dateOfBirth && formatDate(dateOfBirth),
-        applicationDate: createdAt.toLocaleDateString('en-US', {
+        applicationDate: createdAt?.toLocaleDateString('en-US', {
```

Comment:
Unrelated to this PR but ran into an error where exporting the applications requests csv would fail if application date checkbox wasn't selected
![image](https://user-images.githubusercontent.com/51224641/152694793-09e72976-6671-4441-8652-e7791ac82376.png)